### PR TITLE
tsx translated

### PR DESCRIPTION
**Pull request by ainhoami**

### CODE REVIEW

File: src/components/languageDropdown.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const LanguageDropdown = () => {
         padding="7px 0"
       >
         <Text>
-        {intl.formatMessage({ id: 'languageDropdown.text' })}
+          {intl.formatMessage({ id: 'languageDropdown.text' })}
```
Comment (shicholas) [inline src/components/languageDropdown.tsx:55]: running the `yarn lint --fix` command should fix, and my other formatting issues, this automatically

**Comment by shicholas** [comment]
fyi @ainhoami when you see the following, you can merge `development` from the NeonLaw repo into yours so your branch is up-to-date.

Screen Shot 2020-05-12 at 8 47 59 AM

**Comment by shicholas** [comment]
Hey @ainhoami. This is almost done. I think the only change you need to make to get the test to pass is this one - 

https://github.com/NeonLaw/interface/blob/1437d6e29938b1e5cb9d00865852117228eaa842/cypress/integration/accessibility.spec.js#L13

Instead of `Switch to ` with the space, please remove the final space and make it `Switch to`.

**Comment by shicholas** [comment]
@ainhoami thanks!

**Comment by ainhoami** [comment]
> @ainhoami thanks!

@shicholas Thank you!

**Pull request merged**